### ISSUE TITLE
Stubs don't seem to take effect in VSCode (PyLance)

### ISSUE THREAD

**Issue by friedererdmann** [LumaPictures/cg-stubs#1]
Hey! I'm super excited about this first release and tested it with VSCode. Sadly I didn't get the code completion to work and wanted to open an issue to see if I'm doing something wrong, more configuration is needed or if something else is a-miss. Thank you!

- I've created a brand new virtual environment from Python 3.9 on Windows
- I've installed `usd-core` and `types-usd`
- Created a VSCode workspace utilizing the virtual environment and set its language server to Pylance.
- Result: I'm not seeing any code completion on any snippet. Example:
```python
from pxr import Usd

Usd.#[No suggestions]
```

VSCode version info
Version: 1.79.2 (user setup)
Commit: 695af097c7bd098fbf017ce3ac85e09bbc5dda06
Date: 2023-06-14T08:57:04.379Z
Electron: 22.5.7
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.19044
PyLance: v2023.6.40

**Comment by chadrik** [comment]
I'm pretty new to VSCode, but I had a quick play with this.  

I had the same problem as you, but I discovered a workaround: you have to change the style of your import:

```python
import pxr.Usd as Usd
```

<img width="635" alt="image" src="https://github.com/LumaPictures/cg-stubs/assets/63251/bdb619bd-c64f-4cfe-9cb0-8a8f284f2db7">

That's pretty annoying.  

It looks like it gets tripped up by the imports in `pxr-stubs/__init__.pyi`

If I change this:

```python
import pxr.Ar as Ar
import pxr.CameraUtil as CameraUtil
import pxr.Garch as Garch
import pxr.GeomUtil as GeomUtil
...
```

to this:

```python
from . import Ar
from . import CameraUtil
from . import Garch
from . import GeomUtil
...
```

then pylance is happy.

I'll release a new version of the stubs with this fix.

**Comment by chadrik** [comment]
I released version 23.5.1 with a fix.

**Comment by friedererdmann** [comment]
Can confirm - works like a charm! Thank you so much!
